NanaZip.MigrateLegacyStringResources/NanaZip.MigrateLegacyStringResources.csproj

```diff
@@ -16,4 +16,8 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Include="$(VSWhereDir)vswhere.exe" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" />
```

Comment:
Maybe this change should be in another PR, lol.

Reply:
Done with #678